### DISCUSSION TITLE
How to run Ultralytics object detection and segmentation models in a few lines of code

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#14043] [Show and tell]
Our new blogpost by [Nicolai Nielsen](https://github.com/niconielsen32) is a step-by-step guide on how to run Ultralytics object detection and segmentation models in a few lines of code. From detecting to segmenting, this blog will take you through all you need to know about how to use Ultralytics YOLOv5 and YOLOv8 as well as other models supported by Ultralytics.

🔎 Key Highlights:

✅ Set up and use Ultralytics YOLOv8 for real-time detection
✅ Run live webcam detection
✅ Explore other models supported by Ultralytics like YOLO-NAS, and transformer-based RT-DETR

Learn more ➡️ https://www.ultralytics.com/blog/running-ultralytics-object-detection-and-segmentation-models-in-a-few-lines-of-code

Join our GitHub Discussions to share your thoughts and learn more from the community!

[![Episode 11 Run all Ultralytics YOLO object detection models in a few lines of code](https://github.com/ultralytics/ultralytics/assets/155453586/df8a77d8-50c8-48a7-8659-9b9c88c31e15)](https://www.ultralytics.com/blog/running-ultralytics-object-detection-and-segmentation-models-in-a-few-lines-of-code)

**Comment by glenn-jocher** [comment]
@NLadi thank you for sharing this valuable resource! The blog post by Nicolai Nielsen is indeed a fantastic guide for anyone looking to get started with Ultralytics' object detection and segmentation models. For those who are new to YOLOv8 or other models like YOLO-NAS and RT-DETR, this step-by-step guide will be incredibly helpful.

For those interested in running YOLOv8 on AzureML, you can follow our [AzureML Quickstart Guide](https://docs.ultralytics.com/guides/azureml-quickstart/). This guide will help you set up and run YOLOv8 models on AzureML, leveraging the cloud's powerful resources for training and deployment.

If you encounter any issues or have specific questions, feel free to ask here or join our GitHub Discussions to engage with the community. Happy coding! 🚀
